srml/support/procedural/src/construct_runtime/transformations.rs

```diff
@@ -0,0 +1,377 @@
+use super::parse::{ModuleDeclaration, ModulePart, RuntimeDefinition, WhereSection};
```

Comment:
Please move all of this into the `mod.rs` file.
Having these `transformation` file is somewhat useless.